examples/v3.0/uspto.yaml

```diff
@@ -0,0 +1,216 @@
+openapi: 3.0.0
+servers:
+  - url: 'https://developer.uspto.gov/ibd-api'
+  - url: 'http://developer.uspto.gov/ibd-api'
+info:
+  description: >-
+    Bulk data search and download API exposes an API to search published patent
+    grants and applications across various fields.
+  version: 1.0.0
+  title: Bulk data search API
+  contact:
+    name: Open Data Portal
+    url: 'https://developer.uspto.gov'
+    email: developer@uspto.gov
+tags:
+  - name: bulkdata
```

Comment:
Consider a description for the tag to show best practices.

Reply:
Will amend.